tests/templates/kuttl/ldap-authentication/02-install-hdfs.yaml.j2

```diff
@@ -0,0 +1,29 @@
+---
+apiVersion: kuttl.dev/v1beta1
+kind: TestStep
+metadata:
+  name: druid-hdfs
+timeout: 600
+---
+apiVersion: hdfs.stackable.tech/v1alpha1
+kind: HdfsCluster
+metadata:
+ name: druid-hdfs
+spec:
+ image:
+   productVersion: "{{ test_scenario['values']['hadoop'].split('-stackable')[0] }}"
```

Comment:
I would suggest to create and us `hadoop-latest` here to avoid unnecessary testcases

Reply:
The "hadoop" dimension in the test definition already only has only one entry, just as "opa". Both missing "-latest" in their name. Is this something which could be checked in general?

Reply:
The idea is that `druid` or `hadoop` can contain a list of versions to tests against.
The `XXX-latest`only contains the latest version we currently support. In general we use the -latest version if the version isn't to important for the actual test. This helps us to reduce the number of testcases and save time and costs.
In this case i would suggest to create `hadoop-latest` (currently being the same as hadoop) as further hadoop versions can be added in the future

Reply:
I see, so it's a future-work-saving measure. I'll add it to the PR.